electron_app/src/DBManager.js

```diff
@@ -121,7 +133,19 @@ const filterUniqueContacts = contacts => {
 };
 
 const getAllContacts = () => {
-  return db.select('name', 'email').from(Table.CONTACT);
+  return db
+    .select('name', 'email')
+    .from(Table.CONTACT)
+    .orderBy('score', 'DESC')
+    .orderBy('name');
+};
+
+const getContactByEmail = (email, trx) => {
```

Comment:
Can you reuse `getContactByEmails()` and pass an one-element array?
Check if you must modify select params